pkg/libvmi/storage.go

```diff
@@ -111,6 +111,13 @@ func WithEphemeralCDRom(cdRomName string, bus v1.DiskBus, claimName string) Opti
 	}
 }
 
+// WithFilesystemDevice adds a generic filesystem
+func WithFilesystemDevice(deviceName string) Option {
```

Comment:
Not sure this one fits the other options, as it leaves it up to the user to define a backing storage. Maybe move this to `vm_test.go` as an unexported option for now?

Reply:
Sounds good! Done.